core/drivers/cdns_uart.c

```diff
@@ -107,6 +108,7 @@ static const struct serial_ops cdns_uart_ops = {
 	.have_rx_data = cdns_uart_have_rx_data,
 	.putc = cdns_uart_putc,
 };
+KEEP_PAGER(cdns_uart_ops);
```

Comment:
The console code needs only `.putc()` and `.flush()`, does this means we should be good with:
```C
static const struct serial_ops cdns_uart_ops = {
	.flush = cdns_uart_flush,
	.getchar = cdns_uart_getchar,
	.have_rx_data = cdns_uart_have_rx_data,
	.putc = cdns_uart_putc,
} __rodata_unpaged;
KEEP_PAGER(cdns_uart_flush);
KEEP_PAGER(cdns_uart_putc);
```
(and same for the other drivers)?